src/osm_configurator/model/project/calculation/file_deletion.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from pathlib import Path
+import os
+
+
+class FileDeletion:
+    """
+    Responsible for deleting files.
```

Comment:
maybe better documentation, what excatly it does, idk

Reply:
genaueres steht ja in der Methode. Ich habs extra offen gelassen, damit man da noch Methoden hinzufügen kann